src/pybind/mgr/dashboard/requirements-lint.txt

```diff
@@ -9,3 +9,4 @@ flake8-colors==0.1.6; python_version >= '3'
 rstcheck==3.3.1; python_version >= '3'
 autopep8; python_version >= '3'
 pyfakefs; python_version >= '3'
+isort==5.5.3
```

Comment:
Just noticed we have `flake8-import-order` commented.
Any reason we are using `isort` over the other?

Reply:
`flake8-typing-imports` is only for `typing` imports, `isort` will fix all import statements.

Reply:
But `flake8-import-order` might be an alternative, need to check that.

Reply:
The question here is, do we only want to warn that the import order is incorrect, then `flake8-import-order` is the way to go; if we want to change the order automatically, then `isort` is the tool of choice. I prefer automatic order because with warnings only we need to fix all of them manually.

Reply:
We should have both the warning and auto-fix, so `isort` is the one to go with.
Please remove `flake8-import-order` from the list as we no longer will need it.